src/components/PrivateRoute/PrivateRoute.jsx

```diff
@@ -5,7 +5,19 @@ import { useUser } from '../../context/UserContext';
 // screen if you're not yet authenticated.
 export default function PrivateRoute({ children, ...rest }) {
   // TODO: Use the user in context to determine whether to redirect to /login
-  // const { user } = useUser();
+  const { user } = useUser();
+  console.log(user, 'USER');
```

Comment:
Friendly neighborhood reminder to please remove old/unnecessary console logs before pushing your code.